TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Program.cs

```diff
@@ -137,6 +139,12 @@
     };
 
     builder.Services.AddTransient<ServiceClient>(_ => serviceClient.Clone());
+    builder.Services
```

Comment:
Can we either chain all of these `Add*` calls together or have separate statements for each?